src/algebra/lie/nilpotent.lean

```diff
@@ -78,3 +78,69 @@ begin
   use k, rw ← le_bot_iff at h ⊢,
   exact le_trans (lie_module.derived_series_le_lower_central_series R L k) h,
 end
+
+section nilpotent_algebras
+
+-- TODO Generalise the below to Lie modules if / when we define morphisms, equivs of Lie modules
```

Comment:
Just out of curiosity... is `lie_module_hom` already on your todo list somewhere?

Reply:
Sort of.

Note that we [already have](https://github.com/leanprover-community/mathlib/blob/391c90a3dcddb923d75eeef62a6958193d9b3e88/src/algebra/lie/basic.lean#L312) `lie_module_hom` but only for two modules over the same Lie algebra.

However to achieve the unification I'm alluding to in this comment, we'd need to generalise this to something like:
```lean
structure lie_module_hom (f : L →ₗ⁅R⁆ L') extends M →ₗ[R] M' :=
(map_lie' : ∀ {x : L} {m : M}, to_fun ⁅x, m⁆ = ⁅f x, to_fun m⁆)
```
where we have `[lie_module R L M]` and `[lie_module R L' M']` etc.

I probably will get round to this at some point but at the moment it's not on my short-term plan.

Reply:
Ooh, sorry, I'm just being sleepy. Yes, this version doesn't seem extremely high prio.

Reply:
I still agree it's worth doing eventually!